scripts/cloud-api-demo/README.md

```diff
@@ -237,4 +237,3 @@ echo "0 * * * * talawa-api python3 reset_database.py --mongo-container develop-m
 ### 8.3 Logging for cron jobs
 
 This will set up logging for the cron jobs and manage log rotation using logrotate.
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.